src/components/SystemPages/Log.vue

```diff
@@ -5,7 +5,11 @@
       <div class="pageContent">
         <h2 class="pageTitle">Adventure Logs</h2>
         <a :href="reverseLink" class="switchOrder">Reverse order</a>
-        <div class="logItems" v-html="log" />
+        <div class="logItems">
+          <template v-for="page in log">
+            {{page.date}} - <a :href="page.href">{{page.title}}</a><br/>
```

Comment:
Still somewhat concerned about performance on this. When I have time, I'm gonna check it out on a few slower laptops to see how it scales.

Reply:
Rendering the log takes about 1 second on my machine, and half of that is just rendering the very large DOM. I will look into getting vue to cache some results so sorting is faster though.